src/john.c

```diff
@@ -430,6 +436,10 @@ static void john_register_all(void)
 	john_register_one(&fmt_crypt);
 #endif
 
+	/* Do we have --format=LIST? If so, re-build fmt_list from it, in requested order. */
+	if (format_list_string && !fmt_check_custom_list(format_list_string))
+		error_msg("Could not parse format list '%s'\n", format_list_string);
```

Comment:
This should actually be dead code, 440 can't be true. But nevermind, it's good measure I guess.